modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GeneNameDescProjection_vertebrates_conf.pm

```diff
@@ -88,15 +88,15 @@ sub default_options {
       },
       {
         source                 => 'danio_rerio',
-        taxons                 => ['Neopterygii','Cyclostomata','Coelacanthimorpha'],
+        taxons                 => ['Actinopterygii','Cyclostomata','Coelacanthimorpha','Chondrichthyes'],
         antispecies            => ['danio_rerio'],
         gene_name_source       => ['ZFIN_ID'],
         project_trans_names    => 1,
         homology_types_allowed => ['ortholog_one2one','ortholog_one2many'],
       },
       {
         source                 => 'homo_sapiens',
-        taxons                 => ['Neopterygii','Cyclostomata'],
+        taxons                 => ['Actinopterygii','Cyclostomata','Coelacanthimorpha','Chondrichthyes'],
```

Comment:
I think it was a deliberate choice to not project from human to coelacanth, presumably the latter being so remote that it didn't make sense.

Reply:
You are right James, thanks for spotting that. Fixed in new commit